diesel_tests/tests/load_extension.rs

```diff
@@ -0,0 +1,30 @@
+use diesel::{sql_query, Connection, RunQueryDsl, SqliteConnection};
+
+fn conn() -> SqliteConnection {
```

Comment:
There is already such a function somewhere in diesel_tests

Reply:
Fixed in b43a91f3e0fa8f23d710adf9d6b98460040e2c31